docs/projects/fps/tutorial.md

```diff
@@ -356,12 +341,13 @@ Our aim is to have health packs which restore lost health to players. So what ha
 * You added an instance of the health pack entity type to the snapshot so it will be present in the world when the game begins.
 * You associated a local representation with your new SpatialOS entity so that Unity will know how to visually represent any health pack it encounters.
 
-You can launch a local deployment of your updated game world from the SpatialOS menu within the Unity editor by clicking **"Local launch"**. This will open a terminal that should tell you when the world is ready.
+You can launch a local deployment of your updated game world from the SpatialOS menu within the Unity editor by clicking **"Local launch"** (`Ctrl + L`). This will open a terminal that should tell you when the world is ready.
 
 Once the world is ready you can:
 
 * View all entities in the inspector from your browser: http://localhost:21000/inspector/
-* Click Play in your Unity editor (if you have the `FPS-Development` scene open) to play the game.
+* Open the `FPS-Development` scene in the Unity editor. The scene file is located in `Assets > Fps > Scene`.
```

Comment:
we highlight menu items differently. use that:
**Assets** > **Fps** > **Scene**

Reply:
Done!